### PR TITLE
New POST endpoint: Send invite emails to users

### PR DESCRIPTION
**Pull request by Ludikrous**
# Sends emails to new users, with an invite link

## To Do:
- secret management for sendgrid api key
  - could use this: https://aws.amazon.com/secrets-manager/
  - or, set up github actions, and add an env var every deploy? might actually be easier to set up

### CODE REVIEW

**Comment by burbelohugo** [comment]
@Ludikrous amazing work!

Let's just add the sendgrid API key manually for now. I will make a ticket to add/retrieve it from AWS secrets manager, hopefully I can get that done this week or next.

**Comment by burbelohugo** [comment]
Let me know when you've added the tests and we can merge, looks great 🎉

File: backend/users/serverless.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ provider:
   stage: ${opt:stage, 'dev'}
   environment:
     USERS_TABLE: ${file(./config/${self:provider.stage}.json):USERS_TABLE}
+    INVITES_TABLE: ${file(./config/${self:provider.stage}.json):INVITES_TABLE}
+    INVITE_TEMPLATE_ID: 'd-ad25e68e009e405eb2b71364a8e25722'
```
Comment (burbelohugo) [inline backend/users/serverless.yml:10]: Can we move these to the config file instead of hardcoding them?

File: backend/users/serverless.yml [hunk 2]
```diff
@@ -6,6 +6,9 @@ provider:
...
+    BASE_INVITE_URL: 'http://technica-platform.s3-website-us-east-1.amazonaws.com/authenticate'
```
Comment (burbelohugo) [inline backend/users/serverless.yml:11]: Please use the `platform.gotechnica.org` url instead, thanks!

**Comment by Ludikrous** [comment]
> Let me know when you've added the tests and we can merge, looks great

@burbelohugo all ready!

**Pull request merged**